cmd/golang.go

```diff
@@ -0,0 +1,102 @@
+package cmd
+
+import (
+	"github.com/spf13/cobra"
+	"fmt"
+	"os"
+	"text/template"
+	"path/filepath"
+	"strings"
+	"path"
+	"net/http"
+	"io/ioutil"
+)
+
+type TemplatingInfo struct {
+	Name string
+	Path string
+}
+
+func newGolangCommand(opts *dobiOptions) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "golang",
+		Short: "Run the remove action for all resources",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			gopath := os.Getenv("GOPATH")
```

Comment:
I like to keep the anonymous run functions really short. It also helps with indentation and keeping shorter lines.

Could you have this call a top level function (like the other commands) ?